lib/manifest.ps1

```diff
@@ -22,7 +30,15 @@ function url_manifest($url) {
         throw
     }
     if(!$str) { return $null }
-    $str | convertfrom-json
+    $json = $str | convertfrom-json
```

Comment:
Again manipulating loaded manifest is not good idea.

Just do this in `install_app` function